pkg/api/validation/validation.go

```diff
@@ -30,12 +30,32 @@ import (
 	"github.com/golang/glog"
 )
 
+const minPort int = 1
+const minHostPort int = 0
+const maxPort int = 65536
+
+const QualifiedNameErrorMsg string = "must match regex [" + util.Dns1123SubdomainFmt + " / ] " + util.Dns1123LabelFmt
+const PortNameErrorMsg string = "must be less than %d characters and match regex " + util.Dns1123LabelFmt
```

Comment:
This has a %d in it, but is not used as a printf format string.  I think this can go away and all places where we use it can switch to DnsLabelErrorMsg